lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -43,10 +44,12 @@ def lint_lines(lines)
 
       def generate_errors(class_name, line_number)
         violated_rules(class_name).map do |violated_rule|
-          message = "Deprecated class `%s` detected matching the pattern `%s`. %s#{' ' + @addendum if @addendum}"
+          suggestion = pad_with_left_space(violated_rule[:suggestion])
+          addendum = pad_with_left_space(@addendum)
+          message = "Deprecated class `%s` detected matching the pattern `%s`.%s#{addendum}"
```

Comment:
Ohhh clever... Okay so that will work for the `@addendum` but I'll still have to left pad the `suggestion`.